src/partner-access/partner-access.service.ts

```diff
@@ -129,6 +133,9 @@ export class PartnerAccessService {
         totalTherapySessionsRemaining,
       );
     }
+    if (!partnerAccess.featureLiveChat && hasFeatureLiveChat === false) {
+      updateCrispProfile({ feature_live_chat: false }, user.email);
```

Comment:
what would happen in the case that they never had live chat? i think this logic would then create a crisp profile for them, just to set `feature_live_chat: false` which means they shouldnt have a crisp account. or am i missing something?